go.mod

```diff
@@ -110,7 +110,7 @@ require (
 	github.com/prometheus/client_model v0.4.0
 	github.com/prometheus/common v0.44.0
 	github.com/rancher/aks-operator v1.3.0-rc1
-	github.com/rancher/apiserver v0.0.0-20230831052300-120e615b17ba
+	github.com/rancher/apiserver v0.0.0-20240205154815-a3b9e3721c1b
```

Comment:
Same explanation as https://github.com/rancher/rancher/pull/44384/commits/590ebad2e2695679f3f4009f2c283b980ec28e58#r1483588582. Adding for context.